app/controllers/articles_controller.rb

```diff
@@ -1,9 +1,21 @@
 class ArticlesController < ApplicationController
   before_action :find_article_and_category, except: [:create]
 
+  def new
+    @article = Article.new
+    authorize @article
+  end
+
+  def show
+    @article = Article.find_by(id: params[:id])
```

Comment:
If you have the `before_action :find_article_and_category` you don't need to assign `@article` again, right?

Reply:
Ah good point!